test/hw1/test_task5_maxsub.py

```diff
@@ -0,0 +1,19 @@
+"""Test for task 5."""
+
+from typing import List, NoReturn, Optional
+
+import pytest
+
+from homework.hw1.task5_maxsub import find_maximal_subarray_sum
+
+
+@pytest.mark.parametrize(
+    ("array", "k", "expected_result"),
+    [([1, 2, 3, 4, 5], 1, 5), ([-1, -2, -3], 1, -1), ([-1, -2, 3], 1, 3)],
```

Comment:
For more readability these tuples can be placed on different lines

Reply:
black . formating turns them like this, since they don't exceed max length of line